package.json

```diff
@@ -34,7 +34,7 @@
     "@octokit/webhooks": "13.2.7",
     "@sinclair/typebox": "0.32.23",
     "@supabase/supabase-js": "2.42.0",
-    "@ubiquibot/permit-generation": "1.3.1",
+    "@ubiquibot/permit-generation": "*",
```

Comment:
I am against the use of the wildcard because it will break on major versions. If we want to get updates, I would suggest `1.x` or `^1.3.1`.

Reply:
https://github.com/ubiquibot/conversation-rewards/pull/108/commits/96f66bb99faa449fa3de860ede3db08744a0fdb6